apidoc/Titanium/UI/Padding.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Padding
+summary: Dictionary object of parameters for the padding/insets applied to all kinds of views.
+properties:
+  - name: left
+    type: Number
+    summary: Left padding/inset
+    default: 0
+
+  - name: right
+    type: Number
+    summary: Right padding/inset
+    default: 0
+
+  - name: top
+    type: Number
+    summary: Top padding/inset
+    default: 0
+
+  - name: bottom
+    type: Number
+    summary: Bottom padding/inset
+    default: 0
+
+---
+# TODO: Should we invert the relationship here? (i.e. Padding extends from HorizontalInsets?)
```

Comment:
I'd go this way around. The concept of `excluding` things is just not how inheritance works IMHO.